docs/source/learn/core_notebooks/dims_module.ipynb

```diff
@@ -0,0 +1,2910 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        beta = pm.Deterministic("beta", pmd.concat([beta0, spline_slopes.diff("knot")], dim="knot"))
</pre><p>Does pm.Deterministic support both <code>dim</code> and <code>dims</code>? Is it on purpose?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/7820/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
<p>No, the <code><code>dim</code></code> you're seeing is the one for <code>concat</code> </p>

Reply:
:')